tests/foreman/api/test_activationkey.py

```diff
@@ -10,7 +9,24 @@
 from robottelo.test import APITestCase
 
 
-@ddt
+def _good_max_content_hosts():
+    """Return a generator yielding valid ``max_content_hosts`` values."""
+    return (gen_integer(*limits) for limits in ((1, 20), (10000, 20000)))
```

Comment:
generators generates values on demand, which means less memory usage. But in the example given, using that on python 2 will have no effect since the range function will process and return all elements. With that said in python 2 xrange is a generator and in python 3 range is the same as xrange on python 2.